composer.json

```diff
@@ -48,7 +48,8 @@
         "auto-scripts": {
             "cache:clear": "symfony-cmd",
             "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd",
-            "security-checker security:check": "script"
+            "security-checker security:check": "script",
+            "assets:install %PUBLIC_DIR%": "symfony-cmd"
```

Comment:
I would moved this line before the security-checker

Reply:
done